orc8r/gateway/c/common/sentry/SentryWrapper.cpp

```diff
@@ -115,6 +116,14 @@ std::string get_snowflake() {
   return buffer.str();
 }
 
+static sentry_value_t before_send(sentry_value_t event, void* hint,
+                                  void* data) {
+  std::string command = "journalctl -u magma@* -n 1000 > ";
```

Comment:
To get sctpd as well:
```suggestion
  std::string command = "journalctl -u sctpd -u magma@* -n 1000 > ";
```